packages/hoppscotch-sh-admin/sh-admin-subpath-access.Caddyfile

```diff
@@ -0,0 +1,11 @@
+:80 :3100 {
+	handle_path /admin* {
+		root * /site/sh-admin-subpath-access
+		file_server
+		try_files {path} /
+	}
+
+	handle {
+		respond 404
```

Comment:
```suggestion
		respond "Please visit /admin to access the Admin dashboard"
```

Instead of a `404`, we can maybe respond with a message like the above, open to suggestions.